pkg/knuu/knuu.go

```diff
@@ -36,6 +37,11 @@ var (
 
 // Initialize initializes knuu with a unique scope
 func Initialize() error {
+	err := godotenv.Load()
+	if err != nil {
+		return ErrCannotLoadEnv.Wrap(err)
```

Comment:
[question] 
- Does the `.env` file have to exist for all tests?
- if it does not exist will it through an error here?